cvat-canvas/src/typescript/drawHandler.ts

```diff
@@ -192,9 +192,11 @@ export class DrawHandlerImpl implements DrawHandler {
 
             if (this.canceled) return;
             if ((xbr - xtl) * (ybr - ytl) >= consts.AREA_THRESHOLD) {
+                this.drawData.annotation_type = 'Manual';
```

Comment:
Should we change this file?
Everything drawn by a user is manual. Right? It seems we need only once put the type in ``cvat-ui/../../canvas-wrapper.tsx`` in callback ``onShapeDrawn``